types/vis/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for vis.js 4.17
+// Type definitions for vis.js 4.18
```

Comment:
Based on https://github.com/almende/vis/blob/master/HISTORY.md#2017-01-15-version-4180 this looks OK.